index.html

```diff
@@ -0,0 +1,308 @@
+      },
+    };
+    </script>
+  </head>
+  <body data-cite="credential-management-1 WebCryptoAPI">
+    <section id="abstract">
+      <p>
+        This document standardizes an API solution for requesting digital
+        identity credentials (e.g., a drivers license) or verifying some aspect
+        of a digital identity credential (e.g., proof of age). The API builds
+        on the [[[credential-management-1]]] as a means to request a digital
+        credential from the user agent or underlying platform.
+      </p>
+    </section>
+    <section id="sotd">
+      <p>
+        This is an unofficial proposal.
+      </p>
+    </section>
+    <h2>
+      Introduction
+    </h2>
+    <p>
+      TBW
+    </p>
+    <h2>
+      Model
+    </h2>
+    <dl class="definitions">
+      <dt>
+        <dfn>digital identity credential</dfn>
+      </dt>
+      <dd>
+        A digital identity credential is a digital representation of a physical
+        identity credential (e.g., a drivers license, passport, etc). A digital
+        identity credential represents a set of claims about the subject of the
+        credential (e.g., the holder of the credential). A digital identity
+        credential is issued by an issuer (e.g., a government agency).
+      </dd>
+      <dt>
+        <dfn>Identity credential provider</dfn>
+      </dt>
+      <dd>
+        <p>
+          A credential provider is a an application or OS-level service that
+          presents UI to the user to request a digital identity credentia For
+          example,s a credential provider could be a mobile wallet application
+          that requests a mobile drivers license from the user.
+        </p>
+        <p>
+          A credential provider is responsible for presenting the user with a
+          list of [=digital credential/request protocol=] specific providers
+          (e.g., a list of mobile drivers license or applications that hold
+          driver's licenses).
+        </p>
+      </dd>
+      <dt>
+        <dfn for="digital identity">request protocol</dfn>
+      </dt>
+      <dd>
+        The request protocol is a standardized format/structure to [=request
+        digital identity credentials=]. For example, a request protocol could
+        be a <a>"mdoc"</a> for mobile drivers licenses as per [[?ISO-18013-5]].
+      </dd>
+    </dl>
+    <h2>
+      Scope
+    </h2>
+    <p>
+      The following is in is scope for the this specification:
+    </p>
+    <ul>
+      <li>Requesting a [=digital identity credentials=].
+      </li>
+    </ul>
+    <p>
+      The following is out of scope
+    </p>
+    <ul>
+      <li>Creating, updating, deleting, or otherwise managing [=digital
+      identity credentials=].
+      </li>
+    </ul>
+    <h2>
+      Extensions to the `CredentialsContainer` interface
+    </h2>
+    <pre class="idl">
+    partial interface CredentialsContainer {
+      Promise&lt;DigitalCredential&gt; requestIdentity(IdentityRequestOptions options);
```

Comment:
Reiterating from the call: It may be useful to abstract the DigitalCredential return type here and design an algorithm that both FedCM and DigitalCredential can hook into to provide identities. That way FedCM can intermingle in UI with 3-party ids and also with WebAuthN, but mDLs and WebAuthN can't be requested interchangably, which was the motivating concern for this API shape IIRC.

Reply:
I haven't read the FedCM spec yet... but yeah, so long as this can be a generic thing without a normative requirement on FedCM, then that works. 

I'm imagining `DigitalCredential` (or whatever we end up calling it) will end up in the Credential Management spec eventually.

Reply:
That sounds perfect. That just means FedCM would subclass DigitalCredential, in the same way it currently subclasses Credential and would put itself in the registry.

I agree that the generic `DigitalCredential` (or whatever) would live well in the Credential Management spec.

Reply:
Can we try to agree with the goals here first? 

> both FedCM and DigitalCredential can hook into to provide identities

This matches much of my intuition, and it is what Chrome would like to see this design converge to. 

@marcoscaceres, does this match your intuition?

Reply:
I don't have a strong opinion (because I'm still not very familiar with FedCM). If both specs can use the same thing, then great. However, the problem remains about `.get()` conflating requesting multiple credential types, and it's impossible for an implementation to distinguish (because they are all part of the same `CredentialCreationOptions` dictionary).

In the [Extensions Points](https://www.w3.org/TR/credential-management-1/#implementation-extension) section, the CM spec encourages spec authors to define "a reasonable mechanism for dealing with requesting **credentials of distinct types**", which is what `. requestIdentity()` is trying to do, and then to pitch that to the Web App Sec WG. 

I think we need to document our options here and present our options to the Web App Sec WG to help us decide what the right course of action is here. 

I think we have general agreement that this poses issues:

```JS
const opts = {  publicKey , identity,  providers };
// who wins? what happens? What UI gets shown? 
navigator.credentials.get(opts);
``` 

Which, again, is the core of the problem. 

The `. requestIdentity()` disambiguates the problem by providing a unique method signature, only allowing this new `DigitalCredentialRequestOptions` dictionary, which deliberately does not inherit from `CredentialRequestOptions` to avoid its limitations, while providing the same functionality.

Reply:
Just as tl;dr: let's move this discussion to https://github.com/WICG/digital-credentials/issues/61

Reply:
Ok, I think we arrived at a good conclusion on https://github.com/WICG/digital-credentials/issues/61. This issue here is now resolved from where I stand, and since we are following @bvandersloot-mozilla 's suggestion, I think it would be consistent with his thoughts. @marcoscaceres is there anything else that you think is actionable here?

Reply:
I'm happy to proceed. Let's do additional discussions in a follow up.